roles/sap_ha_install_anydb_ibmdb2/tasks/main.yml

```diff
@@ -0,0 +1,165 @@
+    # Default install file is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/db2installPCMK
+    # Default RPM directory is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/Linux
+    # Default SRPM (Source) directory is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/Linux/SRPMS
+    - name: SAP HA AnyDB - IBM Db2 HADR - Identify IBM Db2 installation media with 'Integrated Linux Pacemaker' RPMs subdirectory
+      ansible.builtin.find:
+        paths: "{{ sap_ha_install_anydb_ibmdb2_software_directory }}"
+        recurse: true
+        file_type: file
+        patterns: db2installPCMK
+        excludes: bin
+      register: __sap_ha_install_anydb_ibmdb2_pcmk
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - List all IBM Db2 'Integrated Linux Pacemaker' RPMs in subdirectory
+      ansible.builtin.find:
+        paths: "{{ (__sap_ha_install_anydb_ibmdb2_pcmk.files[0].path | dirname) + '/Linux/' + ('rhel' if ansible_os_family == 'RedHat' else 'suse' if ansible_os_family == 'Suse') }}"
+        recurse: true
+        file_type: file
+        patterns: "*.rpm"
+      register: __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Install all IBM Db2 'Integrated Linux Pacemaker' RPMs for RHEL
+      ansible.builtin.dnf:
+        name: "{{ __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files.files | map(attribute='path') | list }}"
+        state: present
+        disable_gpg_check: true
+      when: ansible_os_family == 'RedHat'
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Install all IBM Db2 'Integrated Linux Pacemaker' RPMs for SLES
+      community.general.zypper:
+        name: "{{ __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files.files | map(attribute='path') | list }}"
+        state: present
+        disable_gpg_check: true
+      when: ansible_os_family == 'Suse'
+
+    # SAP Note 3100287 - DB6: Db2 Support for Pacemaker Cluster Software
+    - name: SAP HA AnyDB - IBM Db2 HADR - Verify IBM Db2 'Integrated Linux Pacemaker' is installed
+      register: __sap_ha_install_anydb_ibmdb2_pcmk_version_check
+      ansible.builtin.shell: |
+        rpm -q corosync | grep -I db2pcmk
+        rpm -q pacemaker | grep -I db2pcmk
+        rpm -q crmsh | grep -I db2pcmk
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Display IBM Db2 'Integrated Linux Pacemaker' installation details
+      ansible.builtin.debug:
+        msg: "{{ __sap_ha_install_anydb_ibmdb2_pcmk_version_check.stdout_lines }}"
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Identify IBM Db2 cluster manager (db2cm) utility
+      ansible.builtin.find:
+        paths: /db2
+        recurse: true
+        file_type: file
+        patterns: db2cm
+      register: __sap_anydb_ibmdb2_db2cm
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Ensure directory for db2cm binary
+      ansible.builtin.file:
+        path: /db2/db2{{ sap_ha_install_anydb_ibmdb2_sid | lower }}/sqllib/bin
+        mode: "0755"
```

Comment:
maybe add owner and group for clarity?

Reply:
Owner will always be root, it's run as root and the binary can only be used by root user. Group can be automatically determined.